source/rust_verify/src/lifetime_generate.rs

```diff
@@ -464,11 +466,19 @@ fn erase_ty<'tcx>(ctxt: &Context<'tcx>, state: &mut State, ty: &Ty<'tcx>) -> Typ
             let assoc_item = ctxt.tcx.associated_item(t.def_id);
             let name = state.typ_param(assoc_item.name.to_string(), None);
             let trait_def = ctxt.tcx.generics_of(t.def_id).parent;
-            match (trait_def, t.args.into_type_list(ctxt.tcx)) {
-                (Some(trait_def), typs) if typs.len() >= 1 => {
+            if let Some(trait_def) = trait_def {
+                // REVIEW: is ignoring non-type arguments okay?
```

Comment:
Good point -- it's probably not ok here.  I think this just got copied from rust_to_vir_base, where we ignore the lifetime variables.  So we could elevate this to a TODO.